### PR TITLE
feat(Farms): Stake and unstake LP show USD price

### PR DESCRIPTION
**Pull request by Woody810**
image

### CODE REVIEW

**Comment by ChefMomota** [comment]
Aptos side will always be `0`. Pls check it

**Comment by Woody810** [comment]
> Aptos side will always be `0`. Pls check it

Fixed.

File: packages/uikit/src/widgets/Farm/components/WithdrawModal/index.tsx [hunk 1]
```diff
@@ -40,26 +44,33 @@ const WithdrawModal: React.FC<React.PropsWithChildren<WithdrawModalProps>> = ({
...
 
   const handlePercentInput = useCallback(
     (percent: number) => {
       const totalAmount = fullBalanceNumber
         .dividedBy(100)
         .multipliedBy(percent)
         .toNumber()
-        .toLocaleString("en-US", { maximumFractionDigits: decimals });
+        .toLocaleString("en-US", { maximumFractionDigits: decimals })
+        .replace(/,/g, "");
```
Comment (ChefMomota) [inline packages/uikit/src/widgets/Farm/components/WithdrawModal/index.tsx:69]: `.toLocaleString("en-US", { maximumFractionDigits: decimals })`
`en-US` will not have `,` issue. So no need every place replace it.

File: packages/uikit/src/widgets/Farm/components/DepositModal/index.tsx [hunk 2]
```diff
@@ -113,26 +114,33 @@ const DepositModal: React.FC<React.PropsWithChildren<DepositModalProps>> = ({
   const handleChange = useCallback(
     (e: React.FormEvent<HTMLInputElement>) => {
       if (e.currentTarget.validity.valid) {
-        setVal(e.currentTarget.value.replace(/,/g, "."));
+        const inputVal = e.currentTarget.value.replace(/,/g, ".");
+        setVal(inputVal);
+
+        const inputValUSDPrice = inputVal.trim() === "" ? "0.00" : new BigNumber(inputVal).times(lpPrice).toFixed(2);
```
Comment (ChefMomota) [inline packages/uikit/src/widgets/Farm/components/DepositModal/index.tsx:120]: Why use `trim` in here?

File: packages/uikit/src/widgets/Farm/components/WithdrawModal/index.tsx (hunk 1 above)
Comment (Woody810) [reply packages/uikit/src/widgets/Farm/components/WithdrawModal/index.tsx:69]: image
It's still have `,` , so need to replace👀

File: packages/uikit/src/widgets/Farm/components/DepositModal/index.tsx (hunk 2 above)
Comment (Woody810) [reply packages/uikit/src/widgets/Farm/components/DepositModal/index.tsx:120]: Fixed

**Comment by Chef-Yogi** [comment]
I think we can keep the ","
it seems we keep the "," in other places, and this is helpful for readability

File: packages/uikit/src/widgets/Modal/ModalInput.tsx [hunk 3]
```diff
@@ -135,6 +140,15 @@ const ModalInput: React.FC<React.PropsWithChildren<ModalInputProps>> = ({
...
           </Text>
         </Flex>
+        {valueUSDPrice && (
+          <Flex justifyContent="flex-start" ml="1rem">
+            <Flex maxWidth="200px">
+              <Text fontSize="12px" color="textSubtle">
+                ~{valueUSDPrice} USD
+              </Text>
+            </Flex>
+          </Flex>
```
Comment (ChefMomota) [inline packages/uikit/src/widgets/Modal/ModalInput.tsx:150]: How about use the `Balance` component?

Comment (Woody810) [reply packages/uikit/src/widgets/Modal/ModalInput.tsx]: > It become jumpy😂

**Review by ChefMomota — APPROVED** [review]

**Pull request merged**